testapi/event/1/sessions

```diff
@@ -587,24 +623,28 @@
       "microlocation": null, 
       "speakers": [
         {
-          "id": 1038, 
-          "name": "Florian Bickmeyer"
+          "id": 143, 
+          "name": "Joe McNamee"
+        }, 
+        {
+          "id": 1345, 
+          "name": "Kirsten Fiedler"
         }
       ], 
       "subtitle": null, 
-      "title": "Die Unsichtbaren \u2013 Migration in Europa", 
-      "track": null
+      "title": "States think we're stupid: Internet censorship around Europe since ACTA", 
```

Comment:
Because previously in the sample data all the tracks were empty. But now the API has been modified so the data received is also different. So I changed the test/sample data. Previously, all tracks were null since no value of track was being provided by API. Now that has been fixed.
https://github.com/fossasia/open-event-orga-server/issues/117